### ISSUE TITLE
It is not blocking the use of magic when reaching 0 Spell Point

### ISSUE THREAD

**Issue by Rakarck** [misthero/dnd5e-spellpoints#42]
Even reaching 0 Spell Point the system is allowing the casting of magic

**Comment by misthero** [comment]
do you have this option enabled?

![immagine](https://user-images.githubusercontent.com/3662610/221876424-5de3ff2c-f5fc-4224-9c1e-618123eb2a1f.png)

**Comment by Rakarck** [comment]
![Snap 2023-02-28 at 11 04 01](https://user-images.githubusercontent.com/126578185/221877091-91dcaa8d-2cc8-42d9-aa2e-4341ebf81e29.png)
![Snap 2023-02-28 at 11 04 10](https://user-images.githubusercontent.com/126578185/221877113-31a2928e-e499-44e2-bc42-94f9284cf952.png)

I'm sure this module blocked the use of spells when the Spell Point was at 0 (In my case I use the name Mana) and a few days ago it stopped working.

I did a test with only the module activated, among other tests, and it continued not blocking

**Comment by Rakarck** [comment]
I don't know if there is any relation, but the module's version is at 10.284, mine foundry vtt is at 10.291. 

I'm also using DnD5e - Fifth Edition System 2.1.5

**Comment by misthero** [comment]
you are right, I get the warning about spell points being out but also the chat message about casting the spell, even if it's not spent any slot in my case.

**Comment by misthero** [comment]
please try the latest release and update me if you see any more issue.

thank you

**Comment by Rakarck** [comment]
Problem solved, however, when you use your module with the Midi QOL, the problem reoccurs.

Probably because Midi has updated to meet the DnD5e 2.1.5

I haven't tested it with other modules, but I believe the problem is just with the Midi.

Even before with the Midi disabled, the problem occurred.

**Comment by misthero** [comment]
I tested with Midi enabled and the spell casting is correctly blocked, maybe there is some specific setting in midi QOL I should check?

**Comment by Rakarck** [comment]
I'm going to do a test by disabling some midi features to see what it is and I'll get back to you

**Comment by Rakarck** [comment]
![Snap 2023-03-07 at 18 13 48](https://user-images.githubusercontent.com/126578185/223555022-2c0fe7a2-26e7-4b7a-b197-ef3331b8e9d9.png)

The part of midi QOL that makes blocking your module not work is this one in the print. For the block to work, you cannot select the Auto consume spell slot, but there is a question, is there a way to make the module block the use of magic even with the Auto consume spell slot activated?

**Comment by misthero** [comment]
If the spell dialog doesn't appear the skill point module is not able to block the casting. It will notify you in chat that you cant cast the spell, but also will appear the chat message about the casted spell.

that said everything is possible when developing, what change is how tricky and the amount of time required to do it.

**Comment by Rakarck** [comment]
Alright buddy, thanks for creating this great module!
